Changed service name in "Next Steps"

jamesche75 opened the pull request:
I have done a PR on all the docs (I think) to change the service name to "Web App for Containers". For this PR, I changed the title of those articles in the "Next Steps" section.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jamesche75 committed:
Changed service name in "Next Steps"

I have done a PR on all the docs (I think) to change the service name to "Web App for Containers". For this PR, I changed the title of those articles in the "Next Steps" section.
